ibm/mas_devops/roles/cp4d/defaults/main.yml

```diff
@@ -33,4 +33,4 @@ common_services_channel: "{{ lookup('env', 'COMMON_SERVICES_CHANNEL') | default(
 cpd_operators_namespace: "{{ lookup('env', 'CPD_OPERATORS_NAMESPACE') | default('ibm-cpd-operators', true) }}"
 cpd_instance_namespace: "{{ lookup('env', 'CPD_INSTANCE_NAMESPACE') | default('ibm-cpd', true) }}"
 cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') }}"
-cpd_supported_versions: ['4.0.9', '4.5.0', '4.5.1', '4.5.2', '4.6.0']
+cpd_supported_versions: ['4.0.9', '4.5.0', '4.5.3', '4.6.0', '4.6.3']
```

Comment:
Let's remove 4.0.9 while we are making changes here.  CP4D support is n-0 so no-one should even really be installing 4.5.x at this point, but let's retain n-1 install support in the automation.